mypy_baseline/_error.py

```diff
@@ -20,6 +20,15 @@
     (?:\s\s\[(?P<category>[a-z-]+)\])?
     \s*
 """, re.VERBOSE | re.MULTILINE)
+REX_LINE_COLUMN = re.compile(r"""
```

Comment:
Instead of making a new regex, can we extend the existing one?
```python
REX_LINE = re.compile(r"""
    (?P<path>.+\.pyi?):
    (?P<lineno>[0-9]+):(?:[0-9]+:)?\s
    (?P<severity>[a-z]+):\s
    (?P<message>.+?)
    (?:\s\s\[(?P<category>[a-z-]+)\])?
    \s*
""", re.VERBOSE | re.MULTILINE)
```